data/js/values/content4.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+    var answer = document.querySelector('#answer span');
+    var button = document.querySelector('button');
+
+    addon.port.on('tokens', function(tokens) {
+        for (var i = 0, l = tokens.length; i < l; i++) {
+            // set the token container to active and shows the token
+            document.querySelector('#' + tokens[i]).classList.add('active');
+        }
+    });
+
+    button.addEventListener('click', function() {
+        // highlight the answer
+        answer.classList.add('answer');
```

Comment:
Ex scenarios:
1. User doesn't select any radio button: The correct answer radio-button is changed to selected.
2. User selects a radio button, but it is incorrect: The incorrect radio-button is deselected and the correct one is selected.
3. User selects a radio button, and it is correct: Nothing changes (as far as radio-button selection goes)